applicationinsights.json

```diff
@@ -5,9 +5,13 @@
   "customDimensions": {
     "service.version": "${BUILD_NUMBER}"
   },
+  "instrumentation": {
```

Comment:
BTW we noticed this is very noisy in prod so we should se what we can strip out at somepoint